roles/services/frontend/templates/nginx-frontend.conf.j2

```diff
@@ -20,7 +20,7 @@ server {
     listen 80 default_server;
     listen [::]:80 default_server;
 
-    server_name api.ipfs-search.com;
+    server_name {{ certbot_domains }};
```

Comment:
Doing it this way is not compatible with using certbot for wildcards. Although I think it's a great idea to make this into a variable, I don't think we should be overlapping them implicitly (we can always do that manually though).

I would suggest naming this `api_server_name`.